src/risedevtool/config/src/main.rs

```diff
@@ -136,7 +136,12 @@ to RiseDev directory."
             }
             Self::Dashboard => {
                 "
-Required if you want to build dashboard from source."
+Required if you want to build dashboard from source.
```

Comment:
This is kept but will be passed to the build script instead of being used in RiseDev steps.